### ISSUE TITLE
submission of isotherms

### ISSUE THREAD

**Issue by mpougin** [NIST-ISODB/isotherm-digitizer-panel#81]
I used the NIST digitizer for the first time today. I uploaded the figure and created the data.json and submission.zip file which I saved locally. However, nothing happens when I click on "Submit". Will a request be sent automatically? It would be great if some confirmation appears

**Comment by ltalirz** [comment]
yeah... we originally wanted to implement an automatic creation of a PR to the database.
we should either do that or remove /gray out the submit button for the moment

**Comment by mpougin** [comment]
thanks for your reply @ltalirz. That would be a good idea, because like that it was not clear for me if something went wrong
